### PR TITLE
METRON-521: Stellar function documentation needs grammar/clarity fixes

### PR DESCRIPTION
**Pull request by cestella**
There are some unclear, inconsistent and ungrammatical text in the Stellar function descriptions. This should be corrected.

### CODE REVIEW

File: metron-platform/metron-common/README.md [hunk 1]
```diff
@@ -332,24 +346,24 @@ The following functions are supported:
   * Description: Transforms the first argument to a double precision number
   * Input:
     * input - Object of string or numeric type
-  * Returns: Double
+  * Returns: Double version of the first argument
 * `TO_EPOCH_TIMESTAMP`
-  * Description: Returns the epoch timestamp of the dateTime given the format. If the format does not have a timestamp and you wish to assume a given timestamp, you may specify the timezone optionally.
+  * Description: Returns the epoch timestamp of the dateTime in the specifiedthe format. If the format does not have a timestamp and you wish to assume a given timestamp, you may specify the timezone optionally.
```
Comment (mmiklavc) [inline metron-platform/metron-common/README.md]: specifiedthe -> specified

File: metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/DataStructureFunctions.java [hunk 2]
```diff
@@ -138,9 +138,9 @@ public Object apply(List<Object> args) {
   }
 
   @Stellar(name="IS_EMPTY"
-          , description="Returns true if string or collection is empty and false otherwise"
-          , params = { "input - Object of string or collection type (e.g. list)"}
-          , returns = "Boolean"
+          , description="Returns true if string or collection is empty and false if otherwise."
+          , params = { "input - Object of string or collection type (for example, list)"}
+          , returns = "True if the string or collection is empty and false otherwise."
```
Comment (mmiklavc) [inline metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/DataStructureFunctions.java]: This conflicts with work from https://github.com/apache/incubator-metron/pull/324

File: metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/DateFunctions.java [hunk 3]
```diff
@@ -114,14 +114,14 @@ public static long getEpochTime(String date, String format, Optional<String> tim
    * Stellar Function: TO_EPOCH_TIMESTAMP
    */
   @Stellar( name="TO_EPOCH_TIMESTAMP"
-          , description="Returns the epoch timestamp of the dateTime given the format. " +
+          , description="Returns the epoch timestamp of the dateTime in the specifiedthe format. " +
```
Comment (mmiklavc) [inline metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/DateFunctions.java]: specifiedthe -> specified

**Comment by mmiklavc** [comment]
+1 pending the mentioned corrections.

File: metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/DataStructureFunctions.java (hunk 2 above)
Comment (cestella) [inline metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/DataStructureFunctions.java]: thanks, got it.

File: metron-platform/metron-common/README.md (hunk 1 above)
Comment (cestella) [inline metron-platform/metron-common/README.md]: thanks, fixed.

File: metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/DateFunctions.java (hunk 3 above)
Comment (cestella) [inline metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/DateFunctions.java]: thanks, fixed.

**Comment by ottobackwards** [comment]
+1

**Pull request closed without merging**